### PR TITLE
Feature/integrate kaeltebus app

### PR DESCRIPTION
**Pull request by michaelstenzel**
Can now create, read, update and delete requests.
Added models and test data for vehicles and requests to cms.
Added forms, views and templates for requests.

### CODE REVIEW

File: src/cms/models/requests/request.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+from cms.models.regions.region import Region
+from cms.models.vehicles.vehicle import Vehicle
+
+
+# pylint: disable=too-few-public-methods
+class RequestManager(models.Manager):
+    def get_queryset(self):
+        # only return true Requests
+        return super(RequestManager, self).get_queryset()
+
```
Comment (timobrembeck) [inline src/cms/models/requests/request.py]:
```suggestion
```
I think you can just remove this class, it doesn't seem to do anything different than the default model manager.

File: src/cms/models/requests/request.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    postcode = models.CharField(max_length=10, validators=[
+        RegexValidator(regex='^.{5}$', message='Length has to be 5', code='nomatch')])
+    city = models.CharField(max_length=250, default="Berlin")
+    country = models.CharField(max_length=250, default="Berlin")
+    latitude = models.FloatField(default=0.0, blank=True)
+    longitude = models.FloatField(default=0.0, blank=True)
+
+    # other metadata
+    archived = models.BooleanField(default=False)
+    #objects = RequestManager()
```
Comment (timobrembeck) [inline src/cms/models/requests/request.py]:
```suggestion
```
If you remove the `RequestManager`, you should also remove this line I think.

**Review by timobrembeck — CHANGES_REQUESTED** [review]
I think if you remove the `RequestManager`, it will automatically fix the remaining pylint error.

Apart from that, you will have to resolve the conflicts in `src/cms/templates/_base.html` before this PR can be merged.

File: src/cms/models/requests/request.py (hunk 1 above)
Comment (michaelstenzel) [reply src/cms/models/requests/request.py]: Have now made this change.

File: src/cms/models/requests/request.py (hunk 2 above)
Comment (michaelstenzel) [reply src/cms/models/requests/request.py]: Have now made this change.

**Review by timobrembeck — APPROVED** [review]
Looks good to me :+1:

**Pull request merged**